googleapiclient/http.py

```diff
@@ -38,11 +38,19 @@
 import os
 import random
 import socket
-import ssl
 import sys
 import time
 import uuid
 
+
+try:
+  import ssl
+except ImportError:
+  _ssl_SSLError = object()
+else:
+  _ssl_SSLError = ssl.SSLError
+
```

Comment:
Single blank lines, not double blank lines, above and below this entire added section. It's special, but it's not double-blank-lines special.